frontend/src/components/RecommendationList.jsx

```diff
@@ -1,51 +1,96 @@
 import React, { useState, useEffect } from "react";
 import { Link, useLocation } from "react-router-dom";
 import axios from "axios";
+import { WINE_QUERY_PARAMS, PRICE, VEGAN, FLAVOR_PROFILE } from "../constants";
 
-// comment out the 2 console logs to get a glimpse of what's in there :)
+// take out the query string from the URL and create an object out of those values
+const queryParamsToObject = (params) => {
+  const queryString = new URLSearchParams(params);
+  const queryObject = {};
+  queryString.forEach(function (value, key) {
+    queryObject[key] = value;
+  });
+  return queryObject;
+};
 
 const RecommendationList = () => {
-  const [myData, setMyData] = useState(null);
+  const [winesData, setWinesData] = useState(null);
   const navigationLocation = useLocation();
-  // console.log(navigationLocation);
+  const filters = queryParamsToObject(navigationLocation.search);
 
   useEffect(() => {
-    // forward the values of the form submission and send a get request with them
-    // localhost will be changed, just here while in development
     const getData = async () => {
-      const response = await axios.get("http://localhost:8080/api/wines", {
-        params: navigationLocation.state,
-      });
-      setMyData(response.data);
-      // console.log(response.data);
+      // localhost will be changed, just here while in development
+      const response = await axios.get(
+        process.env.REACT_APP_API_ENDPOINT_GET_WINES,
+        {
+          params: filters,
```

Comment:
eslint may complain that you are using `filters` here and it's not part of the deps array.
In this case you should use `navigationLocation.searc` as deps because it's a string and the reference doesn't change across rerenders, you would just need to call `queryParamsToObject(navigationLocation.search)` here again.